samples/bluetooth/mesh/ble_peripheral_lbs_coex/README.rst

```diff
@@ -0,0 +1,216 @@
+.. _bluetooth_ble_peripheral_lbs_coex:
+
+Bluetooth: Mesh and Peripheral Coexistence
+##########################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Bluetooth® Mesh and Peripheral Coexistence sample demonstrates how to combine Mesh and legacy Bluetooth low energy features in a single application.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf52840dk_nrf52840, nrf52dk_nrf52832, nrf52833dk_nrf52833, nrf52833dk_nrf52820
+
+The sample also requires a smartphone with the following apps:
+  1. Nordic Semiconductor's nRF Mesh mobile app:
+
+     * `nRF Mesh mobile app for Android`_
+     * `nRF Mesh mobile app for iOS`_
+
+  #. `nRF Connect for Mobile`_
+
+
+
+Overview
+********
+
+The purpose of this sample is to showcase an application where a peripheral can operate independently from the mesh.
+It combines the features of the :ref:`peripheral_lbs` and :ref:`bluetooth_mesh_light` samples into a single application.
+The LBS controls the state of a LED and monitors the state of a button on the device. The mesh controls and monitors the
+state of a separate LED on the device. The Mesh and the LBS does not share any state on the device and operates
+independently of each other.
+
+To be truly independent the LBS must be able to advertise its presence without any dependencies on the mesh.
+Since mesh traditionally uses the advertiser as its main channel of communication, the application must be configured to
+allow sharing of this resource. To achieve this, extended advertising is enabled, with two simultaneous advertising sets.
+One of these sets is used to handle all mesh related communication, while the other is used for advertising the LBS.
```

Comment:
```suggestion
To be truly independent, the LBS must be able to advertise its presence independently of mesh.
Since mesh traditionally uses the advertiser as its main channel of communication, the application must be configured to allow sharing of this resource.
To achieve this, extended advertising is enabled, with two simultaneous advertising sets.
One of these sets is used to handle all mesh related communication, while the other is used for advertising the LBS.
```